test/index.js

```diff
@@ -59,7 +59,23 @@ describe('helpers', function() {
                 '/Users/me/Sandbox/app/bb',
                 '/Users/me/Sandbox/app/ab'
             );
-            assert.equal(parentPath, '/Users/me/Sandbox/app/');
+            assert.equal(parentPath, '/Users/me/Sandbox/app');
+
+            // try with absolute paths that have identical chars after ending
+            // directory.
+            var parentPath2 = helpers.findParentPath(
+                '/Users/me/Sandbox/app/node_modules',
+                '/Users/me/Sandbox/app/nero/src'
+            );
+            assert.equal(parentPath2, '/Users/me/Sandbox/app');
```

Comment:
Oh! So the earlier version would match `/Users/me/Sandbox/app/n`?
 Now we ensure we end the match at `/`.